fala/settings/base.py

```diff
@@ -214,6 +214,8 @@ def root(*x):
 
 LAALAA_API_HOST = os.environ.get("LAALAA_API_HOST", None)
 
+FALA_NO_MAP = os.environ.get("FALA_N0_MAP", "").lower() == "true"
```

Comment:
I think you have a zero instead of an O `FALA_N0_MAP`

Reply:
I think it's worth adding prefix `FEATURE_FLAG_` to feature flags, e.g. `FEATURE_FLAG_NO_MAP`, which communicates that it should be removed once the feature is released

Reply:
Nice plan to default it to False

Reply:
I honestly don't know how a zero got in there!